File: src/py/CumulFROM.py
   
cumulTTFromAnthesis = 0
cumulTTFromFlagLeafLiguleJustVisible = 0
cumulTTFromEndGrainFilling = 0
        
if "Anthesis" in calendarMoments:
    if (SwitchMaize == 0): cumulTTFromAnthesis = cumulTT-calendarCumuls[calendarMoments.index("Anthesis")]
     
if "FlagLeafLiguleJustVisible" in calendarMoments:
    if (SwitchMaize == 0): cumulTTFromFlagLeafLiguleJustVisible = cumulTT-calendarCumuls[calendarMoments.index("FlagLeafLiguleJustVisible")]
    
if "EndGrainFilling"in calendarMoments:
    if (SwitchMaize == 0): cumulTTFromEndGrainFilling = cumulTT-calendarCumuls[calendarMoments.index("FlagLeafLiguleJustVisible")]

File: src/cs/CumulFROM.cs

cumulTTFromZC_65 = 0.0D;
cumulTTFromZC_39 = 0.0D;
cumulTTFromZC_91 = 0.0D;
        
if (calendarMoments.Contains("Anthesis")){
    if (SwitchMaize == 0)
        cumulTTFromZC_65 = cumulTT-calendarCumuls[calendarMoments.IndexOf("Anthesis")];
}    
if (calendarMoments.Contains("FlagLeafLiguleJustVisible")){
    if (SwitchMaize == 0)
        cumulTTFromZC_39 = cumulTT-calendarCumuls[calendarMoments.IndexOf("FlagLeafLiguleJustVisible")];
}   
if (calendarMoments.Contains("EndGrainFilling")){
    if (SwitchMaize == 0)
        cumulTTFromZC_91 = cumulTT-calendarCumuls[calendarMoments.IndexOf("FlagLeafLiguleJustVisible")];
}